core/src/executor/context/blend_context.rs

```diff
@@ -3,19 +3,25 @@ use {
     std::{fmt::Debug, rc::Rc},
 };
 
+#[derive(Debug)]
+pub enum BlendContextRow {
+    Single(Option<Row>),
+    Shared(Option<Rc<Row>>),
```

Comment:
Currently we don't have ues case of `BlendContextRow::Shared(None)`.
So, `Shared(Rc<Row>)` would be enough for now.